dataprofiler/tests/profilers/test_unstructured_text_profile.py

```diff
@@ -164,4 +165,95 @@ def test_case_sensitivity(self):
             expected_word_count = {'hello': 1, 'name': 1, 'grant': 2, 'bob': 1,
                                    'friends': 1}
             self.assertDictEqual(expected_word_count, profile['word_count'])
-        
\ No newline at end of file
+
+    def test_text_profile_with_wrong_options(self):
+        with self.assertRaisesRegex(ValueError,
+                "TextProfiler parameter 'options' must be of type"
+                " TextProfilerOptions."):
+            TextProfiler("Name", options="wrong_data_type")
+
+    def test_options_default(self):
```

Comment:
My one concern with these tests is all of them are single samples. if we could break them up such that they are multiple samples, would be better,
i.e. just turn something like `pd.Series(["This is test, a Test sentence.!!!"])` -> `pd.Series(["This is test,", " a Test sentence.!!!"])`

Reply:
added the case with two samples